### PR TITLE
chores(doc): change proposal for ActionDropdown

### PR DESCRIPTION
**Pull request by acateland**
**What is the problem this PR is trying to solve?**
Using for the first time the component and using the doc is stumbled on some comprehension issues
**What is the chosen solution to this problem?**
update the doc to take into account this issue, removing the less interesting part and making some other more concise and clear.

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [x] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

**Comment by jmfrancois** [comment]
Nice ! I would add a link to the react-bootstrap dropdown documentation

File: packages/components/src/Actions/ActionDropdown/ActionDropdown.component.md [hunk 1]
```diff
@@ -1,176 +1,115 @@
...
+| divider       | Bool                                                                                       |
+| withMenuItem  | Bool                                                                                       |
+| menuItemProps | [react-bootstrap](https://react-bootstrap.github.io/components/dropdowns/#menu-item-props) |
+| liProps       | Object                                                                                     |
 
-The ```<li>``` and ```<MenuItem>``` can be deeply customized with menuItemProps and liProps.
+Your component will be wrapped into a `MenuItem` if `withMenuItem` property is set to true
+This will give you access to customisation of [react-bootstrap](https://react-bootstrap.github.io/components/dropdowns/#menu-items) `MenuItem` component
+
+The `<li>` and `<MenuItem>` can be deeply customized with menuItemProps and liProps.
```
Comment (nicomaligne) [inline packages/components/src/Actions/ActionDropdown/ActionDropdown.component.md]: With the LI we are loosing some stuff from react bootstrap like the accessibility with the keyboard shortcut for example.
If we decide to keep it like that, you can maybe precise it in the documentation.

**Comment by nicomaligne** [comment]
I will use this doc as a reference for my next one, thanx Axcel ;)

File: packages/components/src/Actions/ActionDropdown/ActionDropdown.component.md [hunk 2]
```diff
@@ -1,176 +1,115 @@
...
+| ----- | ---------------------------------- |
+| items | Required (array OR Immutable.List) |
+
+each composed item being an object composed of
+
+| Properties | type            |
+| ---------- | --------------- |
+| label      | Required String |
+| hidelabel  | Bool            |
+| icons      | String          |
```
Comment (jmfrancois) [inline packages/components/src/Actions/ActionDropdown/ActionDropdown.component.md]: I don't see the divider case, so label is not always required

File: packages/components/src/Actions/ActionDropdown/ActionDropdown.component.md [hunk 3]
```diff
@@ -1,176 +1,115 @@
...
-	beforeItemsDropdown: [] or {},
-	itemsDropdown: [] or {},
-	afterItemsDropdown: [] or {},
-}
-``` 
-They are 3 'placeholders' in the component to inject new stuff.
-'itemsDropdown' is the classic content list of the dropdown, equivalent to where the items props will be rendered.
-And you have a 'beforeItemsDropdown' and 'afterItemsDropdown' key, that will be rendered before or after the itemsDropdown in the list, if you need specific stuff that will give you more flexibility.
 
+Using this you can defined additionnal component that your `ActionDropdown` will show.
```
Comment (jmfrancois) [inline packages/components/src/Actions/ActionDropdown/ActionDropdown.component.md]: Using this **props**
I would add a link to Inject documentation here

File: packages/components/src/Actions/ActionDropdown/ActionDropdown.component.md [hunk 4]
```diff
@@ -185,34 +124,35 @@ components: {
 	],
 }
 ```
+
 #### 4 ) overlay
+
 You can have a overlay / tooltip on the ActionDropdown.
```
Comment (jmfrancois) [inline packages/components/src/Actions/ActionDropdown/ActionDropdown.component.md]: either a link or an example would be nice here.

File: packages/components/src/Actions/ActionDropdown/ActionDropdown.component.md [hunk 5]
```diff
@@ -185,34 +124,35 @@ components: {
...
 
 #### 5 ) select callback
+
 With the onSelect props you can add a callback when an item is selected.
```
Comment (jmfrancois) [inline packages/components/src/Actions/ActionDropdown/ActionDropdown.component.md]: same here

File: packages/components/src/Actions/ActionDropdown/ActionDropdown.component.md [hunk 6]
```diff
@@ -1,176 +1,115 @@
-
-
 # ACTION DROPDOWN COMPONENT
```
Comment (jmfrancois) [inline packages/components/src/Actions/ActionDropdown/ActionDropdown.component.md]: Maybe the real name would be better: ActionDropdown component

**Pull request merged**